modules/ROOT/pages/ign-storage.adoc

```diff
@@ -114,6 +114,43 @@ storage:
 <4> We use the previously defined partitions as devices in a RAID1 md array.
 <5> The resulting md array is used to create an EXT4 file system.
 
+.Example for adding a /var partition to the primary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.0.0
+storage:
+  disks:
+  - device: /dev/vda <1>
+    wipe_table: false <2>
+    partitions:
+    - number: 5 <3>
+      size_mib: 0
+      start_mib: 0
```

Comment:
Should we mention how to leave space for expanding the root partition?

Reply:
Not sure. I think we should discuss the strategy further before encoding it here.